apps/api/src/app/events/usecases/update-job-status/update-job-status.usecase.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { JobRepository } from '@novu/dal';
+
+import { UpdateJobStatusCommand } from './update-job-status.command';
+
+@Injectable()
+export class UpdateJobStatus {
+  constructor(private jobRepository: JobRepository) {}
+
+  public async execute(command: UpdateJobStatusCommand): Promise<void> {
+    await this.jobRepository.updateStatus(command.organizationId, command._jobId, command.status);
```

Comment:
Someone would say it is ridiculous to have this small use case... I'd rather have this simple and functionalities separated (and above all, out of the workflow queue service) and be able to easily modify them as needed.

Reply:
I think that's totally fine here.